.github/workflows/os_java_ci.yml

```diff
@@ -56,12 +56,19 @@ jobs:
           path: ~/.m2
           key: ${{ runner.os }}-${{ matrix.java-version }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-m2
+      - name: Set FLOW_TYPE to pull-request
+        if: github.event_name == 'pull_request'
+        run: echo "FLOW_TYPE=pull-request" >> $GITHUB_ENV
+      - name: Set FLOW_TYPE to branch
+        if: github.event_name == 'push'
+        run: echo "FLOW_TYPE=branch" >> $GITHUB_ENV
       - name: Build Chain ${{ matrix.java-version }}
         id: build-chain
-        uses: kiegroup/github-action-build-chain@v2.5.7
+        uses: kiegroup/github-action-build-chain@v2.6
```

Comment:
How about we make this just `2`?
It seems excessive to have to bump this for every little change.

Reply:
you are right, but I would treat build-chain version strategy in a different PR.

Reply:
OK.